docs/calib.rst

```diff
@@ -0,0 +1,141 @@
+	\end{eqnarray}
+
+where :math:`(\Delta x, \Delta y)` are the :term:`wedge offsets`.
+
+
+Spectral Dispersion
+^^^^^^^^^^^^^^^^^^^
+The :term:`spectral dispersion` describes the wavelength along the spectral trace.  The spectral dispersion of a :term:`grism` element is often constant with wavelength, which corresponds to a linear (or low-order polynomial) in the parameter:
+
+.. math::
+	\lambda(t;x_0,y_0) = \alpha_0(x_0,y_0) + \alpha_1(x_0,y_0)\,t + \alpha_2(x_0,y_0)\,t^2 + \ldots
+
+which will be given by a :class:`~slitlessutils.core.wfss.config.StandardPolynomial`.  However, :term:`prism` elements often exhibit a dispersion that is highly non-linear (as a function of wavelength), which can be described as a `Laurent polynomial <https://mathworld.wolfram.com/LaurentPolynomial.html>`_ (e.g. `Bohlin et al 2000 <https://www.stsci.edu/files/live/sites/www/files/home/hst/instrumentation/acs/documentation/instrument-science-reports-isrs/_documents/isr0001.pdf>`_):
+
+.. math::
+	\lambda(t;x_0,y_0) = \beta_0(x_0,y_0) + \frac{\beta_1(x_0,y_0)}{(t-t^*(x_0,y_0))} + \frac{\beta_2(x_0,y_0)}{(t-t^*(x_0,y_0))^2} + \frac{\beta_3(x_0,y_0)}{(t-t^*(x_0,y_0))^3} + \ldots
+
+This introduces an additional parameter :math:`t^*` that effectively modulates the non-linearity.  This form is implemented in the :class:`~slitlessutils.core.wfss.config.ReciprocalPolynomial()` object.
+
+
+
+.. note::
+	This formulation of the spectral trace and dispersion differs from what was used by `aXe <https://hstaxe.readthedocs.io/en/latest/>`_, which effectively employed the path length along the trace as the parameter.  However, this formulation is more computationally efficient with no loss in accuracy.  
+
+
+
+Field-Dependence
+^^^^^^^^^^^^^^^^
+
+As noted above, the coefficients in the trace and dispersion polynomials can be polynomials of the undispersed positions:
+
+.. math::
+	\kappa(x_0,y_0) = \kappa_{0,0} + \kappa_{1,0}\,x_0 + \kappa_{0,1}\,y_0 + \kappa_{2,0}\,x_0^2 + \kappa_{1,1}\,x_0\,y_0 + \kappa_{0,2}\,y_0^2 + \ldots
+
+where :math:`\kappa` can be any of the elements of :math:`a, b, \alpha`, :math:`\beta`, or :math:`t^*`. These spatial polynomials are specified by :class:`~slitlessutils.core.wfss.config.SpatialPolynomial` and are of fixed total order :math:`n`.  This implies the number of any set of these coefficients will be a `triangular number <https://en.wikipedia.org/wiki/Triangular_number>`_ and serialized with `Cantor pairing <https://en.wikipedia.org/wiki/Pairing_function>`_.  
+
+
+.. note::
+	In all above cases, the coefficients :math:`{a}, {b}, {\alpha}` will be unique for each spectral order, 
+	must be determined from calibration observations, and 
+
+
+Usual Workflow
+^^^^^^^^^^^^^^
+
+Since ``slitlessutils`` is largely predicated forward-modeling the WFSS data, the usual workflow begins with a known direct image position and assumed wavelength, then the WFSS image position can be found by:
```

Comment:
```suggestion
Since ``slitlessutils`` is largely predicated on forward-modeling the WFSS data, the usual workflow begins with a known direct image position and assumed wavelength. The WFSS image position can then be found by:
```
Maybe this reads a little better?

Reply:
I toyed around with things, and I can't come up with anything better.